microsoftgraph/microsoft-graph-explorer-v4
# Fix: Response area container heights

## Overview
 
Fixes #2066 
The page doesn't move around as much when selecting response area tabs

## Testing Instructions

* Load GE on your local environment
* Select and Run any sample query e.g GET /me
* Select different Response Area Tabs 
* Observe consistent height when there is data 
* Observe consistent height when no data is present
